addon_service/authorized_storage_account/serializers.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, *args, **kwargs):
         view_name=view_names.detail_view(RESOURCE_TYPE), required=False
     )
     display_name = serializers.CharField(
-        allow_blank=True, required=False, max_length=256
+        allow_blank=True, allow_null=True, required=False, max_length=256
```

Comment:
nit: is there a reason for this to be nullable vs having a default `""`? i.e. do we need to distinguish between the differing empty values?

Reply:
i encountered the frontend sending `display_name: null`, which i think would error without `allow_null=True` (tho could also talk about a frontend fix)

Reply:
I missed that this was in the serializer, so I retract my complaint. That said, I *think* that `required=False` will just not load it into `validated_data` if it's `None`? I know that `required=False` won't include a `None` value in the serialized output, but not sure if that behavior is bi-directional...

Reply:
i think it's [different on the way in](https://github.com/encode/django-rest-framework/blob/2da473c8c8e024e80c13a624782f1da6272812da/rest_framework/fields.py#L515-L524) -- `required=False` allows missing keys, but `None` is rejected unless `allow_none=True`